modules/ROOT/pages/apikit-4-display-fault-task.adoc

```diff
@@ -1,28 +1,34 @@
-= To Display a Typed Fault
+= Display a Typed Fault
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In this procedure, you make the ListInventory operation display the typed fault. For the tshirt2.wsdl file, there is just one typed fault named TshirtFault, which is mapped to every operation that the WSDL has: OrderTshirt, ListInventory, and TrackOrder.
+Make the `ListInventory` operation display the typed fault. In the tshirt2.wsdl file, there is one typed fault named *TshirtFault*, which is mapped to the operations that the WSDL has:
+
+* `OrderTshirt`
+* `ListInventory`
+* `TrackOrder`
 
 APIkit for SOAP supports the following types of SOAP faults:
 
 * Soap Fault 1.1
 * Soap Fault 1.2
 
-You can select either type using any WSDL file, even for those files that do not contain typed faults.
+You can select either type by using any WSDL file, even for files that do not contain typed faults.
 
-. In Studio, search for fault in the palette.
+. In Anypoint Studio, search for *fault* in the command palette.
 +
-Multiple SOAP Fault components might appear.
+Multiple `SOAP Fault` components can appear.
```

Comment:
Is this even necessary?

Reply:
This is relevant because some `SOAP Fault`components may trigger an error and the user should select another SOAP Fault component.
Please, let me know if you think we can do without this sentence.